search/search_repository.go

```diff
@@ -685,6 +693,22 @@ func (r *GormSearchRepository) listItemsFromDB(ctx context.Context, criteria cri
 		return nil, 0, errors.NewBadParameterError("expression", criteria)
 	}
 
+	if childIterations {
+		itrRepo := iteration.NewIterationRepository(r.db)
+
+		where += fmt.Sprintf(` AND `)
+		allitrs := []string{}
+		for _, itr := range iterations {
+			i, _ := uuid.FromString(itr)
+			citrs, _ := itrRepo.LoadChildren(ctx, i)
```

Comment:
@baijum Please don't skip error handling. We've had issues in past because of this.
Just add a `if err != nil` block after this.

Reply:
Fixed. 0daae67f399af7ab122d760eb4e67a3f807ec94b